packages/lodestar-utils/test/unit/json.test.ts

```diff
@@ -135,6 +165,14 @@ describe("Json helper", () => {
       {id: "object of basic types", json: {a: 1, b: 2}, output: "a=1, b=2"},
       // eslint-disable-next-line quotes
       {id: "object of objects", json: {a: {b: 1}}, output: `a={"b":1}`},
+      {
+        id: "error metadata",
+        json: toJson({
```

Comment:
This is part of the test data, you should not use here the function to be tested